parameters_betabarrel.csv

```diff
@@ -0,0 +1,5 @@
+queue,localhost
```

Comment:
Where is this used? I only see it back in the parameters for the gattaca servers, but we do not have a 'localhost' `queue` nor a 'localhost' `QoS`. You can specify a _localhost_ `backend` instead of _slurm_ for molgenis compute, but that is not what we want on these slurm-in-a-box machines: the jobs sould be submitted to Slurm with sbatch. So I'm a bit confused...

Reply:
check